packages/@aws-cdk/alexa-ask/README.md

```diff
@@ -3,3 +3,55 @@
 ```ts
 const alexaAsk = require('@aws-cdk/alexa-ask');
 ```
+
+### Alexa as deploy target for CodePipeline
+
+You can deploy to Alexa using CodePipeline with the following DeployAction.
+
+```ts
+// Read the secrets from ParameterStore
+const clientId = new cdk.SecretParameter(stack, 'AlexaClientId', {ssmParameter: '/Alexa/ClientId'});
+const clientSecret = new cdk.SecretParameter(stack, 'AlexaClientSecret', {ssmParameter: '/Alexa/ClientSecret'});
+const refreshToken = new cdk.SecretParameter(stack, 'AlexaRefreshToken', {ssmParameter: '/Alexa/RefreshToken'});
+
+// Add deploy action
+new alexa.AlexaSkillDeployAction(stack, 'DeploySkill', {
+  stage: deployStage,
+  runOrder: 1,
+  inputArtifact: sourceAction.outputArtifact,
+  clientId: clientId.value,
+  clientSecret: clientSecret.value,
+  refreshToken: refreshToken.value,
+  skillId: 'amzn1.ask.skill.12345678-1234-1234-1234-123456789012',
+});
+```
+
+If you need manifest overrides you can specify them as `overrideArtifact` in the action.
+
+```ts
+// Deploy some CFN change set and store output
+const executeChangeSetAction = new PipelineExecuteChangeSetAction(this, 'ExecuteChangesTest', {
+  stage: deployStage,
+  runOrder: 2,
+  stackName,
+  changeSetName,
+  outputFileName: 'overrides.json',
+  outputArtifactName: 'CloudFormation',
+});
+
+const clientId = new SecretParameter(this, 'AlexaClientId', {ssmParameter: '/Alexa/ClientId'});
+const clientSecret = new SecretParameter(this, 'AlexaClientSecret', {ssmParameter: '/Alexa/ClientSecret'});
+const refreshToken = new SecretParameter(this, 'AlexaRefreshToken', {ssmParameter: '/Alexa/RefreshToken'});
```

Comment:
I don't think you have to repeat these here - they're the same as above, so they don't really bring new information. I would just delete these.